### ISSUE TITLE
Pay Outstanding Invoices

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/hq#300]
[Paying Invoices](https://docs.google.com/spreadsheets/d/1RYoaLBfHFi5KH8XoKfM55VEmlkeEyn2Mws_X4ya_PiU/edit#gid=1142431849)

**Comment by markwilliamfirth** [comment]
Require approvals from @iteles

**Comment by nelsonic** [comment]
A _bit_ more detail in this issue would be _really_ good. 
![pay-invoices-issue-description](https://cloud.githubusercontent.com/assets/194400/26151960/eeb7b042-3afc-11e7-9401-b6422adf21a0.png)
What I would _expect_ to see:
+ [ ] _**Precise**_ description of the issue/task to be completed. _Specifically_:
  + [ ] _Which_ invoices need to be paid? (_without giving out any personally identifiable information_)
  + [ ] How _many_ are there?
  + [ ] What is the total amount of the invoices? Do we have enough cash in the bank to cover payments?
  + [ ] Are any invoices "_over due_"? (_or are we paying proactively_?)
+ [ ] If all invoices are in a _system_ describe the system. or link to the _process/checklist_ where the steps have been described. We do _have_ a well-defined process, _right_? 🤔 
+ [ ] ***Estimate*** the time required to make the payments (_add a **label** to the issue_)
+ [ ] Do the work of paying invoices.
+ [ ] Assign the issue to @iteles  if it requires approval.
+ [ ] Confirm the original estimate was accurate.
+ [ ] Close the issue.

The **`#OneJob`** of "Ops" is to describe _all_ tasks in sufficient detail so they can be performed by someone `else` and _crucially_ ***re-performed*** without further instruction.
Unless we are developing a ["Pit Crew"](https://www.amazon.co.uk/d/Books/Checklist-Manifesto-How-Things-Right-Atul-Gawande/1846683149) (_i.e. **comprehensive repeatable checklists**_) we are _destined_ for _sad/wasted times_.

**Comment by markwilliamfirth** [comment]
@nelsonic 

> Which invoices need to be paid? (without giving out any personally identifiable information)

The invoices can't be identified on GitHub without giving out sensitive information unless you want us to reference all invoices that need to be paid by timestamp at which they came in

>How many are there?

This changes too frequently to update, the number fluctuates constantly as invoices are paid and as they come in

> What is the total amount of the invoices? Do we have enough cash in the bank to cover payments?

This is sensitive information that @iteles doesn't wish to share https://github.com/dwyl/hq/issues/321#issuecomment-301476335

>Are any invoices "over due"? (or are we paying proactively?)

We pay before the due date

>If all invoices are in a system describe the system. or link to the process/checklist where the steps have been described. We do have a well-defined process, right? 🤔

Invoices are stored in our emails. They come into the finance email, @iteles approves the invoices by emailing the finance email #312 . I add a `to pay` label. I pay them and then move them into a `paid` label.

>Estimate the time required to make the payments (add a label to the issue)

This is dependent on the number of invoices. We proactively pay as we go along. I only created this issue as we'd developed a backlog and it needed to be accounted for in our weekly priorities. In this case it probably took around 25m.

>The #OneJob of "Ops" is to describe all tasks in sufficient detail so they can be performed by someone else and crucially re-performed without further instruction.

At the moment I'm a one-man team, so this is not currently a priority, but it will be as soon as I am managing an ops team.

>Unless we are developing a "Pit Crew" (i.e. comprehensive repeatable checklists) we are destined for sad/wasted times.

We proactively pay as we go along. I only created this issue as we'd developed a backlog and it needed to be accounted for in our weekly priorities. In this case it probably took around 25m

**Comment by iteles** [comment]
Note: Invoices should no longer be 'saved in emails'. Since the beginning of this financial year they should now be saved to Xero on a rolling weekly basis as they are being paid so bank reconciliation is always up to date and we have a clear picture of our financial situation at the end of every week.

**Comment by markwilliamfirth** [comment]
@iteles agreed that this process will change once our bookkeeping is done

**Comment by iteles** [comment]
@markwilliamfirth I am just conjecturing that the reason this issue is assigned to me is because there are invoices that need approving?

**Comment by iteles** [comment]
dwyler invoice spreadsheet updated with approvals and comments.
Column added to track corresponding client invoices.
All client invoices updated and approved.

**Comment by iteles** [comment]
Superseded by #411
